userguide/tasks.rst

```diff
@@ -88,7 +89,7 @@ lists the configurable options for a cron job.
    |                     |                             |                                                                                                         |
    +---------------------+-----------------------------+---------------------------------------------------------------------------------------------------------+
    | Schedule a Cron Job | drop-down menu              | Select how often to run the cron job. Choices are *Hourly*, *Daily*, *Weekly*, *Monthly*, or *Custom*.  |
-   |                     |                             | Select *Custom* to open the advanced scheduler.                                                         |
+   |                     |                             | Select *Custom* to open the advanced scheduler. Spaces are not allowed in the custom scheduler.         |
```

Comment:
This is better, but still not specific.  Are spaces not allowed in the name, in the time values, where?